config.json

```diff
@@ -647,6 +647,18 @@
       ]
     },
     {
+      "uuid": "e2b2654f-0f8c-1580-35d1-06a7173a61b9b66d44d",
```

Comment:
the last portion is too long (should be 12 hexadecimal digits but has 19 by my count), so this is not a valid https://en.wikipedia.org/wiki/Universally_unique_identifier (I am assuming this is what a UUID is)

Reply:
Sorry. I generated this with [configlet](https://github.com/exercism/configlet/), but I hadn't seen that [it might not generate valid UUIDs](https://github.com/exercism/configlet/issues/99). This should be fixed now.